JavaAssignments/src/saket/Primenumber.java

```diff
@@ -0,0 +1,38 @@
+package saket;
+
+public class Primenumber {
+
+	// static boolean flag = true;
+
+	public static void main(String[] a) {
+
+		int startnum = Integer.parseInt(a[0]);
+		int Endnum = Integer.parseInt(a[1]);
+		int sum = 0;
+		for (int i = startnum; i <= Endnum; i++) {
+			boolean b = prime(i);
+			if (b == true) {
+				System.out.println("Prime number between " + startnum + " to " + Endnum + " " + i);
```

Comment:
Printing message is not correct. for every prime number it will say "Prime number between startnumber to endnum 
 3". [where i = 3]

you should print better message like , "3 is prime number" and so on.

at the last should say "Sum of all prime numbers --> 1234"